src/main/webapp/app/admin/science/admin-science.component.html

```
@@ -0,0 +1,127 @@
+<div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)